docs/experimental-module-policy.md

```diff
@@ -0,0 +1,35 @@
+# Drupal Core Experimental module policy
+
+Drupal 8 core introduced the concept of [experimental modules][1].
+These are modules that are provided with Drupal core for testing purposes, but that are not yet fully supported.
+They offer a wide range of functionalities, from migration management
+to site layout management.
+
+Due to the experimental nature of the modules from the OpenEuropa platform,
+we need to define a set of policies on their use inside OpenEuropa Components.
+
+## Minimum stability required
+
+Currently the experimental modules follow the different stability levels defined by Drupal:
+alpha, beta, RC and stable.
+
+In order for OpenEuropa to provide a stable enough set of components, only experimental modules
+in **beta state or greater** will be allowed to be used.
+
+This is done because modules in beta state have a stable API that will only be changed
+in very extreme circumstances. See the official [documentation][2] for more information on the subject.
```

Comment:
> in very extreme circumstances

Does this mean that they will only be changed while standing on thin polar ice, on one leg, using Word as an IDE, on a hijacked mobile connection? :grin: 

In fact when something is in beta the API is considered stable without exceptions. Whenever an API change is done, great care is taken to provide a backwards compatibility layer with deprecated functions that ensure the API remains intact until developers find the time to update their code to use the new API. This part of the sentence can be omitted.

Reply:
I'll take part of your comment and add it to the documentation because I think its quite useful